contributing/ORIGIN_TRIALS.md

```diff
@@ -0,0 +1,22 @@
+# Origin trials
+Origin trials enable developers to use an experimental feature in production and provide essential feedback.
+
+The AMP Origin Trial process is inspired by the [Chromium Origin Trial process](https://github.com/GoogleChrome/OriginTrials). 
+
+Traditionally, a feature in experimental mode can be used in development, but cannot be pushed to production. With Origin trials, interested developers can opt-in to a test an experimental feature in production, with the following expectations:
+- The test is for a limited time.
+- The feature will likely undergo some changes after an Origin Trials.
+- Origin trials present an opportunity to implement and benefit from a new feature before it’s fully live. The feature will live on the developer's site, rather than guarded by an experiment, and feedback can directly influence the direction of the feature.
+
+## When to consider an Origin Trial
```

Comment:
Should we add some guidance on "are you sure you need an origin trial"?

Some starting thoughts:
- Not every experiment will need one.
- Your experiment is feature complete with a stable API.
- You're not sure this API is the "right" one and want to avoid potential versioning pain.
- Data from opt-in experiments are insufficient and you need data from real-world users.
- The potential versioning pain is much greater than the pain of origin trial logistics.

Reply:
Added inline.